src/dataset-builder/DatasetBuilder.ts

```diff
@@ -386,7 +386,7 @@ export const ConceptSetSelector = ({
     onChange,
```

Comment:
I think we want to get the correct "shopping cart" behavior for this UI, where we include concepts that are already selected as selected in the new `concept-set-creator` mode. In the case of adding concept criteria, each concept is added as an independent criteria so it doesn't really make sense to show already selected items in the concept selector view. But in this case, I think the selector view should show all already selected items, and let the user both select and de-select items. I'm not exactly sure how the UX works though; we could pre-populate the cart and the UI's commit operation would overwrite the existing selection with the new selection. Although, we'd want to be careful about preserving any selected value sets from a previously selected concept.